spec/lib/reports/brexit_subscribers_report_spec.rb

```diff
@@ -0,0 +1,32 @@
+RSpec.describe Reports::BrexitSubscribersReport do
+  describe "#call" do
+    let!(:date) { Time.zone.local(2020, 7, 12) }
+    let(:list) { create(:subscriber_list, :brexit, title: "brexit", slug: "/brexit") }
```

Comment:
Since we only use the :brexit trait once I think it'd be preferable to set the tags here rather than creating a new trait. It'd save having to look up what the trait means when trying to understand this code.